modules/cli/internal/ui/statestore.go

```diff
@@ -26,6 +26,17 @@ func NewStateStore() *StateStore {
 	}
 }
 
+// BuildStateStore creates a new StateStore with the given parameters
+func BuildStateStore(isAutoScroll, isWordWrap, isBorderless bool) *StateStore {
```

Comment:
I think this should be named `NewStateStore` and the one without arguments `DefaultStateStore`, like you have `DefaultTUIConfig`